projects/core/src/util/try-normalize-http-error.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpErrorResponse } from '@angular/common/http';
+import { LoggerService } from '../logger';
+import { HttpErrorModel } from '../model';
+import { normalizeHttpError } from './normalize-http-error';
+
+export function tryNormalizeHttpError(
```

Comment:
[MINOR]
This is the new suggested API. So let's mark the legacy API `normalizeHttpError` as `@deprecated` in favor of this one - `tryNormalizeHttpError`.
Also let's mention the in the deprecation JSDocs that `normalizeHttpError` will be removed from public API together with removing the feature toggle `ssrStrictErrorHandlingForHttpAndNgrx`.